src/instructlab/model/backends/common.py

```diff
@@ -7,8 +7,22 @@
 import socket
 import typing
 
-# Local
-from ...configuration import get_model_family
+# Third Party
+from instructlab.training.chat_templates import (
+    ibm_generic_tmpl as granite,  # type: ignore[import-untyped]
+)
+from instructlab.training.chat_templates import (
+    ibm_legacy_tmpl as granite_legacy,  # type: ignore
+)
+from instructlab.training.chat_templates import mistral_tmpl as mistral  # type: ignore
+
+# First Party
+from instructlab.common import SupportedModelArchitectures
+from instructlab.configuration import get_model_family
+from instructlab.utils import get_model_arch, get_model_template_from_tokenizer
+
+# mypy: disable_error_code="import-untyped"
```

Comment:
is this necessary?

Reply:
I tried everything before doing this - but it kept complaining about importing chat templates from training